65.md

```diff
@@ -4,73 +4,56 @@ NIP-65
 Relay List Metadata
 -------------------
 
-`draft` `optional` `author:mikedilger`
+`draft` `optional` `author:mikedilger` `author:vitorpamplona`
 
-A special replaceable event meaning "Relay List Metadata" is defined as an event with kind `10002` having a list of `r` tags, one for each relay the author uses to either read or write to.
+Defines a replaceable event using `kind:10002` to advertise preferred relays for discovering a user's content and receiving fresh content from others.
 
-The primary purpose of this relay list is to advertise to others, not for configuring one's client.
+The event MUST include a list of `r` tags with relay URIs and a `read` or `write` marker. If the marker is omitted, the relay is used for both purposes.
 
-The `content` is not used and SHOULD be an empty string.
+The `.content` is not used.
 
-The `r` tags can have a second parameter as either `read` or `write`. If it is omitted, it means the author uses the relay for both purposes.
-
-Clients SHOULD, as with all replaceable events, use only the most recent kind-10002 event they can find.
-
-### The meaning of read and write
-
-Write relays are for events that are intended for anybody (e.g. your followers). Read relays are for events that address a particular person.
+```json
+{
+  "kind": 10002,
+  "tags": [
+    ["r", "wss://alicerelay.example.com"],
+    ["r", "wss://brando-relay.com"],
+    ["r", "wss://expensive-relay.example2.com", "write"],
+    ["r", "wss://nostr-relay.example.com", "read"],
+  ],
+  "content": "",
+  ...other fields
+```
 
-Clients SHOULD write feed-related events created by their user to their user's write relays.
+This NIP doesn't fully replace relay lists in `kind:3`. Instead, `Kind:3` relays SHOULD serve as a fallback option for users that don't have a `kind:10002` event. 
```

Comment:
I don't think it's appropriate to reference kind:3 relay lists as if they are a thing. There is no NIP for them. I would reference it, but amongst a more general "other relay lists".  Otherwise you are putting out a SHOULD that cannot be properly coded against since nobody knows what kind:3 relay lists are (no NIP for them).

So maybe this:  "This NIP doesn't fully replace relay lists that are designed to configure a client's usage of relays (such as kind:3 style relay lists).  Clients MAY substitute other relay lists in situations where a kind:10002 relay list cannot be found."

Reply:
Changed